build.sh

```diff
@@ -299,8 +300,6 @@ case "$COMMAND" in
 
     "osx-swift")
         xcrealmswift "-scheme 'RealmSwift OSX' -configuration $CONFIGURATION build"
-        rm -rf build/osx
```

Comment:
it's done in `osx` which is invoked as target dependency of building `RealmSwift`